R/PIOutputMapping.R

```diff
@@ -108,6 +119,13 @@ PIOutputMapping <- R6::R6Class(
         ))
         private$.observedDataSets[[data[[idx]]$name]] <- data[[idx]]
       }
+      private$.dataCombinedObject$addDataSets(private$.observedDataSets, groups = private$.quantity$path)
```

Comment:
I think all of this must happen in `ParameterIdentification` at time of creation, so we don't need the `DataCombined` object here. Why?

- Adding the data sets here makes the method `removeObservedDataSet()` invalid
- Setting data transformations AFTER adding the data sets will have not effect